praktikumsplaner-frontend/frontend/src/components/assign/PraktikumsstelleCardZuweisung.vue

```diff
@@ -121,6 +131,20 @@ const unassignDialogTitle = ref<string>("Zuweisung aufheben?");
 const unassignConfirmDialog = ref<boolean>(false);
 const warningDialog = ref<boolean>(false);
 const nwkToAssignUnassing = ref<Nwk>();
+
+const properties = defineProps<{
+    modelValue: Praktikumsstelle;
+}>();
+
+const emits = defineEmits<{
+    (e: "updated", praktikumsstelleToUpdate: Praktikumsstelle): void;
```

Comment:
I think for the v-model directive to work it should be update:modelValue

Reply:
In https://github.com/it-at-m/Praktikumsplaner/blob/feature/updatePraktikumsstellen/praktikumsplaner-frontend/frontend/src/components/assign/PraktikumsstellenListZuweisung.vue the binding of model-value should be changed to v-model directive.